kotlinlib/src/mill/kotlinlib/ktlint/KtlintModule.scala

```diff
@@ -57,15 +65,14 @@ object KtlintModule extends ExternalModule with KtlintModule with TaskModule {
 
   /**
    * Reformats Kotlin source files.
-   *
-   * @param check if an exception should be raised when formatting errors are found
-   *              - when set, files are not formatted
    */
   def reformatAll(
-      check: mainargs.Flag = mainargs.Flag(value = false)
+      @arg(positional = true) sources: Tasks[Seq[PathRef]] =
+        Tasks.resolveMainDefault("__.sources")
   ): Command[Unit] = Task.Command {
     ktlintAction(
-      KtlintArgs(format = true, check = check.value),
+      KtlintArgs(format = true, check = true),
```

Comment:
Just a guess, but isn't this a typo? Shouldn't this be
```scala
KtlintArgs(format = true, check = false),
```
@0xnm

Reply:
no, because `check` flag controls exit code handling (see https://github.com/com-lihaoyi/mill/blob/d2df820a88b08933870e299231faa28bb94945ba/kotlinlib/src/mill/kotlinlib/ktlint/KtlintModule.scala#L136-L140). So if format succeeds, exit code will be 0, but we still want to throw exception if something went wrong.

Reply:
Oh, I see, cool, thank :)